org.openfodder.OpenFodder.yml

```
@@ -0,0 +1,65 @@
+app-id: org.openfodder.OpenFodder
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
```

```suggestion
runtime-version: '21.08'
```

consider 20.08 deprecated for new packages.

---

I rely on an older version of LLVM.

---

it doesn't build with llvm12 or 13?

Not sure why llvm11 doesn't come for a more recent runtime, but there is not point in going that route.

---

just to clarify. 20.08 runtime will be EOL in 22.08. 

you should build this with a more recent compiler. Chances that it doesn't support a more recent compiler is a bug.

---

12 and 13 spit out error messages.